src/main/java/com/ghusse/ci/violations/bbcloud/lib/client/Client.java

```diff
@@ -31,24 +32,24 @@ public void setAuthentication(String userName, String password) {
     this.clientV2.setAuthentication(userName, password);
   }
 
-  public List<Comment> listCommentsForPullRequest(PullRequestDescription pullRequestDescription) throws IOException {
+  public List<Comment> listCommentsForPullRequest(PullRequestDescription pullRequestDescription) throws IOException, RestClientException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Refactor this method to throw at most one checked exception instead of: java.io.IOException, com.ghusse.ci.violations.bbcloud.lib.client.implementation.RestClientException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1160)